tests/utils/verification/volumeproperties.go

```diff
@@ -161,3 +164,25 @@ func GetAssociatedPolicyName(hostname string, volName string) (string, error) {
 	}
 	return op, err
 }
+
+// GetVMGroupForVolume returns vmgroup field of volume using admin cli
+// If the volume does not exist, err will be filled with "exit status 1"
+func GetVMGroupForVolume(hostName string, volName string) (string, error) {
+	cmd := admincli.ListVolumes + "-c volume,vmgroup 2>/dev/null | grep " + volName
+	op, err := ssh.InvokeCommand(hostName, cmd)
+	if err != nil {
+		log.Printf("GetVMGroupForVolume return with err: %s", err.Error())
+		return "", err
+	}
+	volProps := strings.Fields(op)
+	if op == "" {
+		msg := fmt.Sprintf("Null value is returned by admin cli when looking for vmgroup field for volume %s", volName)
+		log.Printf(msg)
+		return op, errors.New(msg)
+	}
+	if len(volProps) != 2 {
+		log.Printf("Admin cli output is expected to consist of two elements only - "+
+			"volume name and vmgrop. Actual output %s ", op)
```

Comment:
what happens if the same volume name is on two separate datastores, then we will get two lines (I had this scenario and verification/volumeproperties.go:GetVMAttachedToVolUsingAdminCli() failed).

I'll raise an issue for docker volume inspect to return the vmgroup for a volume so we can use it directly.

I've added a function to return a map of volume properties in PR #1500 and that can be modified to return the vmgroup for a volume.

The above approach is prone to errors in different scenarios and have a lot of this code,  as much as possible should be using docker to get properties filtered out for our use.